### PR TITLE
IGAPP-1314: Adjust title of search results

### PR DESCRIPTION
**Pull request by sarahsporck**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: translations/translations.json [hunk 1]
```diff
@@ -3461,7 +3461,7 @@
       "notAvailable": "Standortbestimmung deaktiviert",
       "timeout": "Standortbestimmung fehlgeschlagen",
       "refresh": "Liste der nahegelegenen Regionen aktualisieren",
-      "pageTitle": "Regionen",
+      "pageTitle": "Lokale Informationen für Dich",
```
Comment (sarahsporck) [inline translations/translations.json:3464]: do I have to delete the translations for the other language, so that this will be translated?

Comment (steffenkleinle) [reply translations/translations.json:3464]: Not sure, I always delete them but if you want to know for sure best ask the service team :)

**Review by steffenkleinle — APPROVED** [review]
Looking good, not tested :)

**Review by LeandraH — APPROVED** [review]
Looks good

**Pull request merged**